### PR TITLE
Documenting potential build issue

### PR DESCRIPTION
**Pull request by dolby360**
Add a note about a potential fix when installing ninja on ubuntu to be compliant with CMake build.

I thought maybe it's better to add another file called SUPPORT.md to gather these bizarre issues and how to solve them.

### CODE REVIEW

**Comment by bhalevy** [comment]
@avikivity who should review this PR?

Also, regarding:
> CI state FAILURE - https://jenkins.scylladb.com/job/releng/job/Scylla-CI/921/

It doesn't seem like any of the failure is related to the documentation change here :)
Just general instability that we need to fix.

File: HACKING.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ required.
 
 Scylla is built with [Ninja](https://ninja-build.org/), a low-level rule-based system. A Python script, `configure.py`, generates a Ninja file (`build.ninja`) based on configuration options.
 
+**Note**: On ubuntu it is possible that the installation directory will be under `/usr/bin/ninja` and CMake search this under `/usr/bin/ninja-build` so a possible sulotion can be using symlink. `ln -s /usr/bin/ninja /usr/bin/ninja-build`
+
```
Comment (avikivity) [inline HACKING.md:55]: I don't see that, installing ninja-build gives me /usr/bin/ninja.

Comment (dolby360) [reply HACKING.md:55]: Have you used apt-get install ninja-build on ubuntu ? I tried it on ubuntu docker.

Comment (avikivity) [reply HACKING.md:55]: yes

Comment (dolby360) [reply HACKING.md:55]: OK then, I'll close this PR. I guess I do something weird.

File: HACKING.md [hunk 2]
```diff
@@ -51,6 +51,8 @@ required.
 
 Scylla is built with [Ninja](https://ninja-build.org/), a low-level rule-based system. A Python script, `configure.py`, generates a Ninja file (`build.ninja`) based on configuration options.
 
+**Note**: On ubuntu it is possible that the installation directory will be under `/usr/bin/ninja` and CMake search this under `/usr/bin/ninja-build` so a possible sulotion can be using symlink. `ln -s /usr/bin/ninja /usr/bin/ninja-build`
```
Comment (nyh) [inline HACKING.md:54]: Regardless of your discussion with Avi below on what actually exists on Ubuntu (which I don't use), I don't understand what this new line is saying. /usr/bin/ninja is not an "installation directory", and our configure.py searches for either "ninja" or "ninja-build" in the user's path, so both should work just fine...

If anything, it's weird that HACKING.md keeps calling ninja with the command "ninja-build", when (as far as I know) it is available as "ninja" on all modern systems. A few years ago on Fedora you had to call it "ninja-build", but this is no longer true.

**Pull request closed without merging**